src/components/Books.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import './styles/Books.css';
+
+const Books = () => (
+  <div className="books">
+    <ul className="booklist">
+      <li className="item">
+        <Book title="The Vicar of Wakefield" author="Oliver Goldsmith" />
+      </li>
+      <li className="item">
+        <Book title="The Decline and Fall of the Roman Empire:" author="Edward Gibbon" />
+      </li>
+      <li className="item">
+        <Book title="The Merchant of Venice" author="Shakespeare" />
+      </li>
+    </ul>
+    <form className="form">
+      <input type="text" placeholder="Book title" className="input" />
+      <input type="text" placeholder="Author" className="input" />
+      <button type="submit" className="form-button">Add Book</button>
+    </form>
+  </div>
+);
+export default Books;
```

Comment:
- [ ] Good job setting your project with react, but it could be better to restructure the `books` component into different components, it is an important philosophy of React and es6 👍  
You could create a <BookList> Renders a list of books using the `<Book>` component  and `<Book>` Takes the book data as a prop and renders the title and the author finally `<NewBookForm>` to  Render a form with inputs for the title and the author of the book

Reply:
@iambenkis I have made the required changes. Kindly have a look at it once and suggest if there is still anything required.